pkg/test/extensions/binary.go

```diff
@@ -1,11 +1,12 @@
-package externalbinary
+package extensions
```

Comment:
Is there a plan for unit/integration tests to be added in a following PR?

Reply:
These code paths are executed by every e2e job that runs on the repo, I hadn't planned for additional integration testing

Reply:
I didn't have anything specific in mind, I just need to understand the standard for testing here. I'll take a closer look at the existing tests.